### DISCUSSION TITLE
Carbon intensity of AI for buildings

### DISCUSSION THREAD

**Discussion by esc-data-science** [alan-turing-institute/ADViCE#93] [Key sector: Built Environment]
Depending on the size and complexity of the model, AI software can be quite carbon-intensive to run. Is that a factor in assessing the overall carbon impact of a heating system?

**Comment by samyoung-dsci** [comment]
Good question. Generative AI can certainly be very carbon intensive due to its energy usage. Other forms of AI may be less carbon intensive, though optimisation algorithms that rely on lots of computation may fall into a similar category.

Accounting for this is very difficult and is unlikely to be easy to factor into the decisions around a heating system (although compute cost, which is linked to emissions, might be). Having said that, because heating is so energy intensive and makes up such a big proportion of our energy usage it seems likely that the emissions reduction from optimisating a domestic heating system is probably much larger than that of the AI used for that purpose.

Interestingly [Heata](https://www.heata.co/housing#Housing_Heata-unit) are trying to actually tie those two things together by installing servers in homes and using the waste heat to heat water for the home! There are also people trying to do that at a larger scale - using waste heat from data centres in district heating networks.
